suitcase/tests/test_spec.py

```diff
@@ -227,7 +214,7 @@ def test_insert_specfile(spec_filename, db_all):
     # Can only insert the spec scans whose type is in the _BLUESKY_PLAN_NAMES
     # list
     scans_expected_to_fail = [scan for scan in specfile if scan.scan_command
-                              not in spec._BLUESKY_PLAN_NAMES]
```

Comment:
This logic seems to be inverted from what it was before?

Reply:
_spec_scan_names is dictionary with keys from  old _BLUESKY_PLAN_NAMES, which is ['dscan', 'ascan', 'ct'].